core/trino-web-ui/pom.xml

```diff
@@ -130,7 +130,7 @@
                         <configuration>
                             <arguments>run flow</arguments>
                             <workingDirectory>src/main/resources/webapp/src</workingDirectory>
-                            <installDirectory>src/main/resources/webapp/dist</installDirectory>
+                            <installDirectory>target</installDirectory>
```

Comment:
Same here

Actually, should this line be removed? It seems like it should use the config from above